extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -51,7 +51,7 @@ describes.realWin('amp-form', {
 
   function getAmpForm(button1 = true, button2 = false, button3 = false,
                       canonical = 'https://example.com/amps.html') {
-    installAmpForm(env.win);
+    new AmpFormService(env.ampdoc);
```

Comment:
See https://github.com/cvializ/amphtml/blob/d2c413e838f3b5e92b842b143243bee934039f90/testing/describes.js#L521

Yeah, just update the comment there.